Sources/SwiftKafka/KafkaClient.swift

```diff
@@ -0,0 +1,69 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the swift-kafka-gsoc open source project
+//
+// Copyright (c) 2022 Apple Inc. and the swift-kafka-gsoc project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of swift-kafka-gsoc project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Crdkafka
+import Logging
+
+/**
+ Base class for ``KafkaProducer`` and ``KafkaConsumer``,
+ which is used to handle the connection to the Kafka ecosystem.
+ */
```

Comment:
This is a super nit but can we use: `///` please instead of `/**`. This aligns with the style we use in almost all Swift on Server repos